tests/testsuite.src/syn_file.at

```diff
@@ -2282,3 +2282,74 @@ AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
 prog.cob:12: error: syntax error, unexpected INPUT, expecting Identifier
 ])
 AT_CLEANUP
+
+# Verify that FD GLOBAL will create default handlers in sub-programs, but
+# only in its own sub-programs
+
+AT_SETUP([Scope of FD GLOBAL in nested programs])
+AT_KEYWORDS([FD GLOBAL nested])
+AT_DATA([prog.cob],[
+ IDENTIFICATION DIVISION.
```

Comment:
Could you try to stick with fixed-form if there's no explicit need for free-form reference format?
The main reason is that it allow to quickly (or at least much more quick) manually cross-test the source with other compilers.

Reply:
ok, they are in fixed form now.